iidm/iidm-xml-converter/src/test/resources/V1_0/remoteReactivePowerControlNotSupported.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure it is necessary to test for all versions until 1.0 ? But it is done now.